api/sendForm.ts

```diff
@@ -0,0 +1,17 @@
+import { IRequest } from "../interface/IRequest";
+import { IResponse } from "../interface/IResponse";
+
+export class SendForm {
+    email: string;
+    constructor(email:string) {
+        this.email = email;
+    }
```

Comment:
Hint: u can use shorthand constructor https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd